docs/source/use_cases/run_pipelines_in_the_cloud/index.rst

```diff
@@ -0,0 +1,95 @@
+==========================================
+Run workflows and common tasks in parallel
+==========================================
+
+.. comment: begin: goto-read-the-docs
+
+.. container:: visible-only-on-github
+
+   +-----------------------------------------------------------------------------------+
+   | **The properly rendered version of this document can be found at Read The Docs.** |
+   |                                                                                   |
+   | **If you are reading this on github, you should instead click** `here`__.         |
+   +-----------------------------------------------------------------------------------+
+
+.. _RenderedVersion: http://googlegenomics.readthedocs.org/en/latest/use_cases/run_pipelines_in_the_cloud/index.html
+
+__ RenderedVersion_
+
+.. comment: end: goto-read-the-docs
+
+Do you have a task that you need to run independently over dozens,
+hundreds, or thousands of files in Google Cloud Storage? The
+`Google Genomics Pipelines API`_ provides an easy way to launch
+and monitor tasks running in the cloud.
+
+  +---------------------------------------------------------------------+
+  | Alpha                                                               |
+  +---------------------------------------------------------------------+
+  | This is an Alpha release of Google Genomics API. This feature might |
+  | be changed in backward-incompatible ways and is not recommended for |
+  | production use. It is not subject to any SLA or deprecation policy. |
+  +---------------------------------------------------------------------+
+
+Overview
+--------
+
+A "pipeline" in its simplest form is simply a task consisting of:
```

Comment:
nit, probably don't need the second simply as it already is in a simplest form :)